docs/apb_cli.md

```diff
@@ -265,63 +299,57 @@ apb broker catalog --name foo-broker
 ```
 
 ---
-### `bundle`
+### `catalog`
 
 ##### Description
-Interact with Ansible Playbook Bundle images present in the `apb` tool
+Interact with OpenShift Service Catalog. Force the Service Catalog to relist it's available APBs from an Ansible Service Broker instance
 
 ##### Usage
 ```bash
-apb bundle [COMMAND] [OPTIONS]
+apb catalog [COMMAND] [OPTIONS]
 ```
 
 ##### Commands
-_deprovision_: Deprovision APB image
-_info_: Print info about APB image
-_list_: List available APB images
-_prepare_: Stamp APB metadata onto Dockerfile in base64 encoding
-_provision_: Provision APB images
+| Subcommand | Description |
+| :---       | :---        |
+| relist     | Force a relist of the OpenShift Service Catalog |
 
 ##### Options
 
 | Option, shorthand  | Description |
 | :---               | :---        |
 | --help, -h         | Show help message |
-| --kubeconfig, -k   | Path to kubeconfig to use |
+| --name, -n         | Name of clusterservicebroker to relist |
 
 
 ##### Examples
-Provision `mediawiki-apb` APB image
+Force a relist of `foo-broker`
```

Comment:
Force a relist of `foo-broker`
->
Force a relist of clusterservicebroker `foo-broker`